app/code/core/Mage/AdminNotification/Model/Inbox.php

```diff
@@ -128,7 +128,7 @@ public function add($severity, $title, $description, $url = '', $isInternal = tr
         if (is_array($description)) {
             $description = '<ul><li>' . implode('</li><li>', $description) . '</li></ul>';
         }
-        $date = date('Y-m-d H:i:s');
+        $date = date(Varien_Db_Adapter_Pdo_Mysql::TIMESTAMP_FORMAT);
```

Comment:
I'd use Varien_Db_Adapter_Pdo_Mysql ::DATETIME_FORMAT since the name is more similar to the meaning, and the constant is the same

Reply:
I thought about those two. You are completely right, there is the third constant that also defines a format and it is the same as  `TIMESTAMP_FORMAT`. I picked `TIMESTAMP_FORMAT` for one main reason: the datrabase columns I checked all have the type **timestamp** not the type **datetime** which both are present in MySQL, are similar but have distinguishing characteristics. I totally think that the format of both will never be different though.

Reply:
wow, they've a timestamp type but the php formatting is YMDIms?? auch